### PR TITLE
Upgrade from 3.1 to 6

### PR DESCRIPTION
**Pull request by kiyros**
upgrade to .NET6,
bug found, edit function does not work in any version (3.1, .net5, .net6)

### CODE REVIEW

**Comment by cengique** [comment]
I see some deprecation warnings:
```
fail: Microsoft.AspNetCore.SpaServices[0]
      (node:1017634) [DEP0111] DeprecationWarning: Access to process.binding('http_parser') is deprecated.
      
fail: Microsoft.AspNetCore.SpaServices[0]
      (Use `node --trace-deprecation ...` to show where the warning was created)
```
@Valteria any reason to worry?

@kiyros I can't get the file upload feature to work, can you confirm?

**Comment by cengique** [comment]
Upload also doesn't work in version 3.1, so needs to be troubleshooted separately. Merging.

File: src/Analysim.Web/Startup.cs [hunk 1]
```diff
@@ -105,8 +105,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, ILoggerM
 
                 if (env.IsDevelopment())
                 {
-                    //spa.UseAngularCliServer(npmScript: "start");
-                    spa.UseProxyToSpaDevelopmentServer("http://localhost:4200");
+                    spa.UseAngularCliServer(npmScript: "start");
```
Comment (cengique) [inline src/Analysim.Web/Startup.cs:108]: @kiyros does this work, should we remove?

**Review by cengique — COMMENTED** [review]
Please check Startup.cs to make sure we accept the change.

File: src/Analysim.Web/Startup.cs (hunk 1 above)
Comment (kiyros) [reply src/Analysim.Web/Startup.cs:108]: it works

**Review by cengique — APPROVED** [review]
confirmed it works

**Pull request merged**